gap/Magic.gd

```diff
@@ -296,6 +289,17 @@
 #!             <Br/>
 #!             <E>Default value: <C>[ ".", "gap", "lib", "examples", "examples/doc" ]</C>.</E>
 #!         </Item>
+#!
+#!         <Mark><A>LaTeXOptions</A></Mark>
```

Comment:
We are kind of split in whether we use `snake_case` or `CamelCase` for option record entries... Well, the former usually were added by @sebasguts and the latter by me :-). It is a bit jarring that we now have `gapdoc.LaTeXOptions` right next to `gapdoc.gap_root_relative_path` ... :-/. But of course this is not new, i.e., not a principle issue with this PR...

IMHO the major argument for using `CamelCase` is that this is what GAP uses, period. Sure, `snake_case` is popular in Python and Julia and other languages -- but not in GAP.

That said, I can also live with use `snake_case`. It would simply be nice if we made these things uniform. Eventually. Not in this PR, just in general :-)

So perhaps we should decide on one of the two conventions, and then convert everything for that (of course while retaining support for the old versions for the foreseeable future, possibly forever).

Reply:
We could actually just support arbitrary versions by converting everything to lower case without underscores.

However, I think GAPDoc likes to call this option record `GAPDocLaTeXOptions` or something like this, so it would make not so much sense to convert it to snake case. If anything, lets just go with camel for making it more gappy.